src/utils/getReactNativeVersion.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ *
+ * @flow
+ */
+const fs = require('fs');
+const path = require('path');
+
+module.exports = function getReactNativeVersion(cwd: string) {
+  try {
+    const pak = JSON.parse(
+      fs.readFileSync(path.join(cwd, 'package.json')).toString()
```

Comment:
You can simplify the code by just `require`'ing the JSON file (it will call `JSON.parse` under the hood):
```js
const pak = require(path.join(cwd, 'package.json'));
```

Reply:
This came from existing https://github.com/callstack/haul/blob/master/src/utils/isReactNativeProject.js do you want me to refactor both?

Reply:
That would be awesome 👍

Reply:
Especially since `require`s are cached

Reply:
Will fix in both